content/tutorial-x-ray-image-processing.ipynb

```diff
@@ -0,0 +1,572 @@
+{
```

Comment:
<p>Suggestion:</p><p> </p><p><code>Image(filename=GIF_PATH, width=400, height=400)</code></p><p> </p><p>Locally, the generated gif was very large and did not fit very nicely in my screen.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
Good idea, done! Hope it renders in a 400x400 box on all machines 😃